config/src/config/consensus_config.rs

```diff
@@ -68,6 +68,10 @@ pub struct ConsensusConfig {
     pub intra_consensus_channel_buffer_size: usize,
     pub quorum_store: QuorumStoreConfig,
     pub vote_back_pressure_limit: u64,
+    /// If backpressure target block size is below it, update `max_txns_to_execute` instead.
+    /// Applied to execution, pipeline and chain health backpressure.
+    /// Needed as we cannot subsplit QS batches.
+    pub min_max_txns_in_block_after_filtering_from_backpressure: u64,
```

Comment:
instead of having max_sending_block_txns_after_filtering_override until 500, and max_txns_from_block_to_execute below 500, in all of the backpressure configs, changing all backpressures to only have one - wanted size, and use this config to decide how to update how proposed block should be created